imports/plugins/core/graphql/server/schemas/shop.js

```diff
@@ -11,6 +11,9 @@ export const typeDefs = `
 
     # Returns a list of groups for the shop with ID \`shopId\`, as a Relay-compatible connection.
     groups(after: ConnectionCursor, before: ConnectionCursor, first: ConnectionLimitInt, last: ConnectionLimitInt, sortOrder: SortOrder = asc, sortBy: GroupSortByField = createdAt): GroupConnection
+
+    # Returns a list of roles for the shop with ID \`shopId\`, as a Relay-compatible connection.
```

Comment:
I know you just copied the comment above, but that one is a bit incorrect. Change both this comment and the one above to say `for this shop` (matching the administrators comment) rather than `for the shop with ID shopId`.